src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationNewWithTernaryOp.java

```diff
@@ -0,0 +1,77 @@
+/* Config:                                                                    //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:   //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ * basicOffset = 4                                                            //indent:1 exp:1
+ * forceStrictCondition = true                                                //indent:1 exp:1
+ * lineWrappingIndentation = 8                                                //indent:1 exp:1
```

Comment:
it is good practice to put in test exact code from user as absolute prove that user case (input+config) is covered.
default is 4 - https://checkstyle.org/config_misc.html#Indentation_Properties

please create one more Input file or find Input file that is used for:
```
        <module name="Indentation">
            <property name="forceStrictCondition" value="true"/>
        </module>
```
and please use exact users code from bug description.

Reply:
done